### ISSUE TITLE
Clean builds have a failing dependency 

### ISSUE THREAD

**Issue by paytonrules** [mParticle/mparticle-android-sdk#78]
It appears that as of this morning you can't import `mparticle:android-core` because `androidx.localbroadcastmanager:localbroadcastmanager` isn't available anymore. Our builds have started failing on account of it. I fixed it our build by using `localbroadcastmanager:1.1.0-alpha` but this is less than an ideal fix. `localbroadcastmanager` is deprecated, so it will need to be resolved as quick as possible.
```
Execution failed for task ':app:dataBindingMergeDependencyArtifactsDebug'.
> Could not resolve all files for configuration ':app:debugRuntimeClasspath'.
   > Could not resolve androidx.localbroadcastmanager:localbroadcastmanager:[1.0.0, ).
     Required by:
         project :app > com.mparticle:android-core:5.33.1
      > Failed to list versions for androidx.localbroadcastmanager:localbroadcastmanager.
         > Unable to load Maven meta-data from https://comscore.bintray.com/Analytics/androidx/localbroadcastmanager/localbroadcastmanager/maven-metadata.xml.
            > Could not get resource 'https://comscore.bintray.com/Analytics/androidx/localbroadcastmanager/localbroadcastmanager/maven-metadata.xml'.
               > Could not GET 'https://comscore.bintray.com/Analytics/androidx/localbroadcastmanager/localbroadcastmanager/maven-metadata.xml'. Received status code 502 from server: Bad Gateway
```

**Comment by anthonylawson** [comment]
Is this because the version in the [POM file](https://search.maven.org/artifact/com.mparticle/android-core/5.33.1/aar) for `androidx.localbroadcastmanager:localbroadcastmanager` is `[1.0.0, )`?

**Comment by paytonrules** [comment]
I don't think so - this error showed up in our CI build this morning, and the POM file hasn't changed.

**Comment by anthonylawson** [comment]
Is interesting that `https://comscore.bintray.com/Analytics` is the repository that it is failing on... Is that an internal repository that proxies Maven Central and Google's Maven Repository?

**Comment by anthonylawson** [comment]
Also, maybe Bintray is not working... Didn't they shut down this past year?

**Comment by samdozor** [comment]
hey all - just verifying - are you using the comscore kit? Our Core SDK should have any reference to Comscore's maven repository. But the Comscore kit does:

https://github.com/mparticle-integrations/mparticle-android-integration-comscore/blob/main/build.gradle#L27

**Comment by paytonrules** [comment]
Yup we are.

**Comment by samdozor** [comment]
@paytonrules have you added that kit as a submodule rather than via Maven? or have you otherwise directly added comscore's maven repo to your maven/gradle build? I'm trying to figure why your project would even be aware of that `https://comscore.bintray.com/Analytics` maven URL

**Comment by paytonrules** [comment]
We added it directly - sorry I can't just give you a gradle file but I think this segment should be non-identifying:

```
    implementation "com.mparticle:android-core:$versions.mparticle"
    implementation "com.mparticle:android-comscore-kit:$versions.mparticle"
    implementation "com.mparticle:android-iterable-kit:$versions.mparticle"
    implementation "com.mparticle:android-adobe-kit:$versions.mparticle"
    implementation "com.mparticle:android-appsflyer-kit:$versions.mparticle"
    implementation "com.mparticle:android-apptentive-kit:$versions.mparticle"
 ```
Should that be changed?

**Comment by alexs-mparticle** [comment]
Tracking internally as 76319

**Comment by einsteinx2** [comment]
We're working on a fix, however we're currently in holiday release blackout, so we will not be updating the Android SDK until January. We will make a best attempt to have this fix included in the first January release.

**Comment by willpassidomo** [comment]
Hey @paytonrules do you have `mavenCentral()` added in your `repositories { }` block? Other than that, I can't imagine what would be causing this outside of a networking issue. 

It _could_ be the case that Gradle is getting confused with the version-range notion since the last few versions of the dependency are postfixed (i.e "-alpha01"). You could test this by excluding the transitive dependency and replacing it with an explicit on, like so:

```
compile ("com.mparticle:android-core:5.33.1"){
        exclude group: 'androidx.localbroadcastmanager', module: 'localbroadcastmanager'
    }       
    compile "androidx.localbroadcastmanager:localbroadcastmanager:1.1.0"
```

Let me know how it goes or if this continues to be an issue :)

**Comment by willpassidomo** [comment]
fixed in version 5.33.2
